lib/provisioning/ProvisionDispatcher.js

```diff
@@ -7,6 +7,8 @@ const zookeeper = require('node-zookeeper-client');
 
 const Logger = require('werelogs').Logger;
 
+const testIsOn = process.env.TEST_SWITCH === '1';
```

Comment:
Same thing, relying on `CI=true` is better as it's consistent with CI's that support OSS.